bin/phastSim

```diff
@@ -142,7 +142,8 @@ if hierarchy:
         infoFile=infoFile,
         verbose=sim_run.args.verbose,
         noNorm=sim_run.args.noNormalization,
-        mutationsTSVinput=sim_run.args.mutationsTSVinput)
+        mutationsTSVinput=sim_run.args.mutationsTSVinput,
+        args = args)
```

Comment:
don't keep this change: instead add an if-else to following from lines 252 depending on whether or not the chromosomeName was supplied: 

```python
# If requested, create a protocol buffer output
if args.createMAT:
    if args.noHierarchy:
        print("Protocol buffer output only supported in hierarchical mode, remove --noHierarchy.")

    else:

        if not args.chromosomeName:
            from Bio import SeqIO
            chromosome = SeqIO.read(args.reference, format='fasta').id

        else:
            chromosome = args.chromosomeName

        genome_tree.write_genome_mat(
            tree=t, 
            output_path=args.outpath, 
            output_file=args.outputFile, 
            chromosome=chromosome
        )
```